robusta_krr/main.py

```diff
@@ -72,6 +72,12 @@ def {func_name}(
                     help="List of namespaces to run on. By default, will run on all namespaces.",
                     rich_help_panel="Kubernetes Settings"
                 ),
+                selector: Optional[str] = typer.Option(
+                    None,
+                    "--selector",
+                    help="Selector (label query) to filter on, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2). Matching objects must satisfy all of the specified label constraints.",
```

Comment:
The `-l` bit is confusing as this is not supported in `krr`.

Reply:
Yeah, help is wrong - I removed `-l`